modules/data.atmosphere/tests/testthat/test.check_met_file.R

```diff
@@ -3,11 +3,8 @@ context("Check met input file correctly detects errors")
 test_that(
   "Check met input correctly finds errors in bad met files",
   {
-    urbana_daily_met <- system.file(
-      "tests/testthat/data/urbana_daily_test.nc",
-      package = "PEcAn.data.atmosphere"
-    )
-    urbana_daily_results <- check_met_input_file(urbana_daily_met)
+
+    urbana_daily_results <- check_met_input_file("data/urbana_daily_test.nc")
```

Comment:
I'm not 100% clear on this and I lean towards saying it *doesn't* always operate relative to the package root. What I'm relying on instead is knowing that the new path here is relative to the working directory: it looks in `modules/data.atmosphere/tests/testthat/data` not `modules/data.atmosphere/data`. My understanding is that this is safe because whatever path the tests are excuted on, we can assume the whole testthat directory will be moved there in one piece.